doc/descriptors.md

```diff
@@ -52,14 +53,20 @@ Descriptors consist of several types of expressions. The top level expression is
 - `raw(HEX)` (top level only): the script whose hex encoding is HEX.
 
 `KEY` expressions:
-- Hex encoded public keys (66 characters starting with `02` or `03`, or 130 characters starting with `04`).
-  - Inside `wpkh` and `wsh`, only compressed public keys are permitted.
-- [WIF](https://en.bitcoin.it/wiki/Wallet_import_format) encoded private keys may be specified instead of the corresponding public key, with the same meaning.
--`xpub` encoded extended public key or `xprv` encoded private key (as defined in [BIP 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)).
-  - Followed by zero or more `/NUM` unhardened and `/NUM'` hardened BIP32 derivation steps.
-  - Optionally followed by a single `/*` or `/*'` final step to denote all (direct) unhardened or hardened children.
-  - The usage of hardened derivation steps requires providing the private key.
-  - Instead of a `'`, the suffix `h` can be used to denote hardened derivation.
+- Optionally, key origin information, consisting of:
+  - An open bracket `[`
+  - 8 hex characters for the fingerprint
```

Comment:
Would it be clearer to replace "fingerprint" with "extended key identifier"? Could also link to https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#key-identifiers if so.

Reply:
It's specifically "The first 32 bits of the identifier are called the key fingerprint.", not the entire identifier.

I think "master fingerprint" is more specific if we're going to change this.

Reply:
It isn't required to be the master fingerprint; just whatever key the device with the private key is on needs to derivate from. I guess most commonly that will be the master, but it could be a child key as well.

Reply:
I've expanded the comment slightly anyway.